LenovoLegionToolkit.WPF/App.xaml.cs

```diff
@@ -72,6 +75,14 @@ private async void Application_Startup(object sender, StartupEventArgs e)
         if (Log.Instance.IsTraceEnabled)
             Log.Instance.Trace($"Flags: {flags}");
 
+        if (flags.QuickActionRunName is not null)
```

Comment:
What is the purpose of this? Is it only so that the CLI can start LLT process if it's not running in background? If yes, I would remove it. It is ok to make it a requirement for LLT to run in bg for CLI to work.

Reply:
I didn't really understand what u meant. I put this if clause here only for judging if this is a normal LLT startup, or requesting for a Quick Action run.